src/core/server/http/cookie_session_storage.ts

```diff
@@ -139,9 +153,26 @@ export async function createCookieSessionStorageFactory<T>(
     path: basePath,
     clearInvalid: false,
     isHttpOnly: true,
-    isSameSite: false,
+    isSameSite: cookieOptions.sameSite === 'None' ? false : cookieOptions.sameSite ?? false,
   });
 
+  // A hack to support SameSite: 'None'.
+  // Remove it after update Hapi to v19 that supports SameSite: 'None' out of the box.
+  if (cookieOptions.sameSite === 'None') {
+    log.debug('Patching Statehood.prepareValue');
+    const originalPrepareValue = Statehood.prepareValue;
```

Comment:
Not documented even it's a public method. Stringifies and encodes a cookie: https://github.com/hapijs/statehood/blob/v6/lib/index.js#L415